src/charm.py

```diff
@@ -207,7 +213,7 @@ def _push_alert_rules(self, alerts: Dict[str, Any]) -> List[str]:
 
     def _ensure_lokitool(self):
         """Copy the `lokitool` binary to the workload container."""
-        if self._nginx_container.exists("/usr/bin/mimirtool"):
+        if self._nginx_container.exists("/usr/bin/lokitool"):
```

Comment:
lol. Really? How did this ever work?

Reply:
it worked because of the following line :eyes: